Packs/FireEyeNX/Integrations/FireEyeNX/FireEyeNX.yml

```diff
@@ -166,15 +164,15 @@ script:
     - default: false
       description: "Searches between two specified time frames. When specifying a\
         \ start_time value, you must specify both a start_time and an end_ time value.\n\
-        Formats: \nYYYY-MM-dd\nYYYY-MM-ddTHH:mm:ss.SSS[+/-]XX:XX"
+        Formats:\nYYYY-MM-dd\nYYYY-MM-ddTHH:mm:ss\nExample:\n 2020-05-01 \n 2020-05-01T00:00:00"
```

Comment:
we also want the option to enter `2 days` or `5 hours` in the start_time and end_time arguments